### PR TITLE
Update aggregate-report to use common credscan yml and update baseline file

### PR DESCRIPTION
**Pull request by sima-zhu**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
This is pre-requisite of PR:
https://github.com/Azure/azure-sdk-tools/pull/2775/files

We are ready to use credscan stable version. Put existing issues into baseline. Team will work on baseline file.
Generate baseline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1376421&view=results
Legacy version: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1376511&view=results
New stable version: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1376784&view=results
If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by weshaggard — APPROVED** [review]
Change looks good but I assume you want to wait for https://github.com/Azure/azure-sdk-tools/pull/2775 and ensure the baseline matches that tool version.

File: eng/common/pipelines/templates/steps/credscan.yml [hunk 1]
```diff
@@ -28,14 +28,14 @@ steps:
 - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
   displayName: CredScan result analysis
   inputs:
-    GdnBreakBaselineFiles: ${{ parameters.BaselineFilePath }}
```
Comment (weshaggard) [inline eng/common/pipelines/templates/steps/credscan.yml:31]: I assume this is for testing and you plan to undo these changes.

**Review by mccoyp — APPROVED** [review]
Thank you for regenerating the baseline, Sima!
